tests/e2e/bin/travis/travis_default-site.conf

```diff
@@ -0,0 +1,43 @@
+# Upstream to abstract backend connection(s) for php
+upstream php {
+        server unix:/tmp/php-cgi.socket;
+        server 127.0.0.1:9000;
+}
+
+server {
```

Comment:
I usually add this to my WP Nginx configs but I don't know if it's really needed here, just letting you know:
```
# deny access to .htaccess files, if Apache's document root concurs with nginx's one
location ~ /\.ht {
  deny all;
}
```

Reply:
Does this make sense if apache is not installed in VM? I'm terrified to dig deeper into that, so happy to add that as is :D

Reply:
My reasoning to have it has been that WP or some plugins sometimes generates that file on their own and on Nginx it would end up being like a text file if I remember correctly. I don't really think it's a concern in controlled E2E environment where you're not installing random plugins. ;-)

Just a heads up that you might wanna check if the file is there, if it has anything sensitive (prolly not even if it's there) and if it's accessible from the internets.

Reply:
Thanks, makes sense :) Incoming